utils/parallel_runner.go

```diff
@@ -0,0 +1,31 @@
+package utils
+
+import (
+	"github.com/jfrog/gofrog/parallel"
+	"sync"
+)
+
+type AuditParallelRunner struct {
+	Runner      parallel.Runner
+	ErrorsQueue chan error
+	Mu          sync.Mutex
```

Comment:
```suggestion
	Mu          sync.Mutex
```
I dont understand what this attribute is, change the name to fit better its usage